ecpy_hqc_legacy/tasks/tasks/instr/views/spdev_views.enaml

```diff
@@ -57,6 +57,14 @@ enamldef DemodSpView(InstrView): view:
         entries_updater << task.list_accessible_database_entries
         tool_tip = EVALUATER_TOOLTIP
 
+    Label: average:
+        text = 'Average'
+    ObjectCombo: average_val:
+        items << ['No avg','Avg before demod','Avg after demod']
+        selected := task.average
+        tool_tip = fill("Check if you want averaged data, "
```

Comment:
done, but cannot get the tool_tip box to be all filled, there are many blank parts